rdtools/filtering.py

```diff
@@ -79,7 +96,80 @@ def tcell_filter(temperature_cell, temperature_cell_low=-50,
             (temperature_cell < temperature_cell_high))
 
 
-def clip_filter(power_ac, quantile=0.98):
+def csi_filter(poa_global_measured, poa_global_clearsky, threshold=0.15):
+    '''
+    Filtering based on clear-sky index (csi)
+
+    Parameters
+    ----------
+    poa_global_measured : pd.Series
+        Plane of array irradiance based on measurments
+    poa_global_clearsky : pd.Series
+        Plane of array irradiance based on a clear sky model
+    threshold : float, default 0.15
+        threshold for filter
+
+    Returns
+    -------
+    pd.Series
+        Boolean Series of whether the clear-sky index is within the threshold
+        around 1.
+    '''
+
+    csi = poa_global_measured / poa_global_clearsky
+    return (csi >= 1.0 - threshold) & (csi <= 1.0 + threshold)
+
+
+def clip_filter(power_ac, model="quantile_clip_filter", **kwargs):
+    """
+    Master wrapper for running one of the desired clipping filters.
+    The default filter run is the quantile clipping filter.
+    
+    Parameters
+    ----------
+    power_ac : pd.Series
+        Pandas time series, representing PV system with
+        a pandas datetime index.
+    model : string, default 'quantile_clip_filter'
```

Comment:
I suggest changing the parameter options to just `quantile`, `xgboost`, and `logic`